<launch>

<arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
<arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />


  <!-- Experiment Playback  700:44sec 600:36sec 400:24sec 200:9sec 0:2sec-->
<param name ="/use_sim_time" value="true"/>  
<node pkg="rosbag" type="play" name="player" output="screen" 
args="--clock -r 0.05 -s 9 -u 7 /mnt/hgfs/Experiment_Data_MIT/point_cloud_2018-05-04-15-42-45.bag"/>

<!-- Crop Image Node -->
<node pkg="opt_flow" type="crop_new.py" name="image_crop1" output="screen">
        <remap from="UBC_Image"  to="/camera/depth/image_raw/compressed" />
	<remap from="RGB_Image"  to="/camera/rgb/image_raw/compressed" />
	<remap from="PointCloud"  to="/camera/depth/points" />
	<param name="ymin" value="33" />
        <param name="ymax" value="270" />
        <param name="xmin" value="125" />
        <param name="xmax" value="372" />
	<param name="pcymin" value="43" />
        <param name="pcymax" value="270" />
        <param name="pcxmin" value="125" />
        <param name="pcxmax" value="372" />
	<param name="boltymin" value="201" />
        <param name="boltymax" value="227" />
        <param name="boltxmin" value="70" />
        <param name="boltxmax" value="102" />
     <rosparam>
     </rosparam>
</node>  

</launch>
